scripts/dev_scripts/integration_tests.sh

```diff
@@ -123,109 +118,45 @@ $RUN_MACARON analyze -rp https://github.com/apache/maven -b master -d 6767f2500f
 
 python $COMPARE_DEPS $DEP_RESULT $DEP_EXPECTED || log_fail
 
-# Analyze micronaut-projects/micronaut-core.
-echo -e "\n=================================================================================="
-echo "Run integration tests with configurations for micronaut-projects/micronaut-core..."
-echo -e "==================================================================================\n"
-DEP_RESULT=$WORKSPACE/output/reports/github_com/micronaut-projects/micronaut-core/dependencies.json
-
-echo -e "\n----------------------------------------------------------------------------------"
-echo "micronaut-projects/micronaut-core: Check the resolved dependency output when automatic dependency resolution is skipped."
-echo -e "----------------------------------------------------------------------------------\n"
-DEP_EXPECTED=$WORKSPACE/tests/dependency_analyzer/expected_results/skipdep_micronaut-projects_micronaut-core.json
-$RUN_MACARON analyze -c $WORKSPACE/tests/dependency_analyzer/configurations/micronaut_core_config.yaml --skip-deps || log_fail
-
-python $COMPARE_DEPS $DEP_RESULT $DEP_EXPECTED || log_fail
-
 # echo -e "\n----------------------------------------------------------------------------------"
-# echo "micronaut-projects/micronaut-core: Check the resolved dependency output with config for cyclonedx gradle plugin (default)."
+# echo "micronaut-projects/micronaut-core: Check the resolved dependency output for cyclonedx gradle plugin (default)."
 # echo -e "----------------------------------------------------------------------------------\n"
+DEP_RESULT=$WORKSPACE/output/reports/github_com/micronaut-projects/micronaut-core/dependencies.json
 DEP_EXPECTED=$WORKSPACE/tests/dependency_analyzer/expected_results/cyclonedx_micronaut-projects_micronaut-core.json
-$RUN_MACARON analyze -c $WORKSPACE/tests/dependency_analyzer/configurations/micronaut_core_config.yaml || log_fail
+$RUN_MACARON analyze -rp https://github.com/micronaut-projects/micronaut-core -b 3.8.x -d 68f9bb0a78fa930865d37fca39252b9ec66e4a43 || log_fail
 
 # TODO: uncomment the test below after resolving https://github.com/oracle/macaron/issues/60.
 # python $COMPARE_DEPS $DEP_RESULT $DEP_EXPECTED || log_fail
 
 echo -e "\n----------------------------------------------------------------------------------"
-echo "micronaut-projects/micronaut-core: Check the e2e output JSON file with config and no dependency analyzing."
+echo "micronaut-projects/micronaut-core: Check the e2e output JSON file with no dependency analyzing."
 echo -e "----------------------------------------------------------------------------------\n"
-JSON_RESULT_DIR=$WORKSPACE/output/reports/github_com/micronaut-projects/micronaut-core/
-JSON_EXPECT_DIR=$WORKSPACE/tests/e2e/expected_results/micronaut-core
-
-declare -a COMPARE_FILES=(
-    "micronaut-core.json"
```

Comment:
Instead of removing this test case, how about using a custom SBOM to analyze selected dependencies?